### PR TITLE
SWS-106 display metrics on service details page

### PR DESCRIPTION
**Pull request by jotak**
- Textual representation only (fancy charts are for later)
- Relaxed some tslint rules

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -22,7 +21,7 @@
         "jsx-no-lambda": false,
         "jsx-no-multiline-js": false,
         "label-position": true,
-        "max-line-length": [ true, 120 ],
+        "max-line-length": [ false, 120 ],
```
Comment (mtho11) [inline tslint.json]: These should probably be in a different PR. But I understand. Believe me ;)

Comment (mtho11) [reply tslint.json]: Right now, I have my own set of tslint.json for development.

File: src/pages/ServiceDetails/ServiceMetrics.tsx [hunk 2]
```diff
@@ -1,18 +1,134 @@
...
   render() {
     return (
       <div>
-        == SERVICE METRICS ({this.props.namespace} / {this.props.service}) ==
+        <h1>
+          == SERVICE METRICS ({this.props.namespace} / {this.props.service}) ==
+        </h1>
+        Range:
+        <select value={this.state.range} onChange={this.onRangeChanged}>
+          <option value="1m">1 minute</option>
```
Comment (mtho11) [inline src/pages/ServiceDetails/ServiceMetrics.tsx]: At some point we should probably make this data driven and not hard coded.

**Comment by jotak** [comment]
@abonas here's a screenshot: img

**Comment by pilhuhn** [comment]
Great @jotak (I personally would more like the bullets be left-aligned, but that is not a blocker or anything).
Perhaps in the next round wrap the sections in some pattern fly box with INPUT/OUTPUT/.. being a header and the bullets being the box content

**Comment by jotak** [comment]
@pilhuhn this is very temporary as the enxt step I think is to replace it with charts, so I didn't even bother in aligning / grouping stuff

**Comment by pilhuhn** [comment]
On 21 Feb 2018, at 16:33, Joel Takvorian wrote:

> @pilhuhn this is very temporary as the enxt step I think is to replace 
> it with charts, so I didn't even bother in aligning / grouping stuff

I think this is already great as a table.

How do I access that page inside the UI ? What is the url?

**Comment by jotak** [comment]
`/namespaces/tutorial/services/preference` but you'd need to fetch that PR server-side, which is not merged: https://github.com/swift-sunshine/swscore/pull/30

File: src/types/Metrics.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export interface MetricValue {
+  Value: number;
+}
+
+export interface MetricHistogram {
+  Average: number;
```
Comment (mtho11) [inline src/types/Metrics.ts]: These should be lower cased No?

File: src/pages/ServiceDetails/ServiceMetrics.tsx (hunk 2 above)
Comment (jotak) [reply src/pages/ServiceDetails/ServiceMetrics.tsx]: @mtho11 I've read your comment too late. You're right, I must bring changes on bother core&ui
